<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeriodForm</class>
 <widget class="QWidget" name="PeriodForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Times New Roman&quot;;
background-color: rgb(50, 50, 50);
color: #b1b1b1;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(93, 93, 93);</string>
       </property>
       <property name="text">
        <string>Введите период</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(60, 60, 60);</string>
         </property>
         <property name="text">
          <string>Начало</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="date_start">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(68, 68, 68);</string>
         </property>
         <property name="minimumDate">
          <date>
           <year>2000</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
         <property name="date">
          <date>
           <year>2023</year>
           <month>10</month>
           <day>27</day>
          </date>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(60, 60, 60);</string>
         </property>
         <property name="text">
          <string>Конец</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="date_end">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(68, 68, 68);</string>
         </property>
         <property name="minimumDate">
          <date>
           <year>2000</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
         <property name="date">
          <date>
           <year>2023</year>
           <month>10</month>
           <day>27</day>
          </date>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="btn_ok">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(75, 75, 75);
	border-radius: 1px;
}
QPushButton:hover {
     background-color: rgb(80,80, 80);
}
QPushButton:pressed {
    background-color: rgb(120,120,120);
}</string>
         </property>
         <property name="text">
          <string>Ок</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_cancel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(75, 75, 75);
	border-radius: 1px;
}
QPushButton:hover {
     background-color: rgb(80,80, 80);
}
QPushButton:pressed {
    background-color: rgb(120,120,120);
}</string>
         </property>
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
